build/marionette.state.js

```diff
@@ -35,7 +39,7 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
     //   preventDestroy: {boolean} If true, then this will not destroy on `component` destroy.
     // }
     constructor: function constructor() {
-      var _ref = arguments[0] === undefined ? {} : arguments[0];
+      var _ref = arguments.length <= 0 || arguments[0] === undefined ? {} : arguments[0];
```

Comment:
Not sure why this line or line 125 are changing, I did not intend for this diff.